UI.tsv

```diff
@@ -518,7 +518,7 @@ UI_20150317_000517	{@st41}Combination
 UI_20150317_000518	{@st43}Quest{/}
 UI_20150317_000519	Quest {@st59b} {nl}{@st59s}- Register to quest notice when there is a new quest {nl} - Yellow for main quest, Blue for sub-quest {nl} - The arrow on the character's foot leads to quest location {nl} - 'Return' icon is displayed when the quest is complete {nl}    (teleport to quest NPC using backspace or mouse click) {nl} - Check the box in front of the quest name to add/remove from the quest notice {/}
 UI_20150317_000520	{@st41b}In Progress{/}
-UI_20150317_000521	{@st41b}Abandon{/}
+UI_20150317_000521	{@st41b}Forfeit{/}
```

Comment:
Wouldn't abandon be the more appropriate term? Forfeit means to give up (something) as a necessary consequence of something else. I don't think we are "forfeiting" a quest as a consequence.